src/server/socketHandlers/mwPresencePublishOut.js

```diff
@@ -0,0 +1,23 @@
+import parseChannel from './parseChannel';
```

Comment:
custom middleware that catches messages before they head out to their respective sockets. I do this to ensure that every message sent is actionable.